src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -307,9 +305,7 @@ const IssueForm = (): JSX.Element | null => {
     };
 
     const handleSelectVaultCheckboxChange = () => {
-      if (!isSelectVaultCheckboxDisabled) {
```

Comment:
This check was redundant, so I removed it. Please correct me if I'm wrong.

Reply:
On a second thought, I decided to revert this change although I had tested it enough. :) Just in case.

Reply:
Done!